### DISCUSSION TITLE
How to minimize the sslocal.exe in the windows10 system tray?

### DISCUSSION THREAD

**Discussion by oneshu** [shadowsocks/shadowsocks-rust#1300] [General]
rt~~
the cmd or powershell can't work.

**Comment by catscarlet** [comment]
Try RBTray.

RBTray is a small Windows program that runs in the background and allows almost any window to be minimized to the system tray by:

- Right-clicking its minimize button
- Shift-right-clicking its title bar
- Using the Control-Alt-Down hotkey

RBTray: https://github.com/benbuck/rbtray


Run sslocal.exe in a cmd, and then minimize it with RBTray.

**Reply by oneshu** [reply]
thank~
Well, I just found that the windows 11 terminal could minmize to the system tray.
![图片](https://github.com/shadowsocks/shadowsocks-rust/assets/45453410/086a1099-0f3a-46d4-97b9-0eab4352a8ee)
